var/spack/repos/builtin/packages/lfortran/package.py

```diff
@@ -23,11 +23,11 @@ class Lfortran(CMakePackage):
 
     depends_on("python@3:", type="build", when="@main")
     depends_on("cmake", type="build")
-    depends_on("llvm@14.0.4", type="build", when="+llvm")
-    depends_on("zlib@1.2.12", type="build")
-    depends_on("re2c@2.2", type="build", when="@main")
-    depends_on("bison@3.4.2", type="build", when="@main")
-    depends_on("binutils@2.38", type="build", when="platform=linux")
+    depends_on("llvm@11:15", type="build", when="+llvm")
+    depends_on("zlib", type="build")
+    depends_on("re2c", type="build", when="@main")
+    depends_on("bison", type="build", when="@main")
```

Comment:
You have to use Bison 3.4, the newer Bison packages have a bug that fails to compile our grammar correctly.

Reply:
Got it!